Makefile

```diff
@@ -0,0 +1,35 @@
+build: ## Build the backend Docker image
```

Comment:
suggestion: add https://github.com/torchbox/wagtailmedia/blob/main/Makefile#L1-L12 to make this self-documenting

Reply:
Love this! Thank you, i've added now